addons/l10n_in_edi/views/account_move_views.xml

```diff
@@ -40,4 +40,17 @@
             </xpath>
         </field>
     </record>
+    <record id="l10n_in_edi_inherit_account_move_search_view" model="ir.ui.view">
+        <field name="name">l10n.in.edi.inherit.account.move.search</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_account_invoice_filter" />
+        <field name="arch" type="xml">
+            <xpath expr="//filter[@name='cancel']" position="after">
+                <separator/>
+                <filter name="l10n_in_edi_to_send" string="To Send for E-Invoice(IN)" domain="[('l10n_in_edi_status', '=', 'to_send')]"/>
```

Comment:
No way to avoid the(IN)  
```suggestion
                <filter name="l10n_in_edi_to_send" string="To Send for Indian E-Invoice" domain="[('l10n_in_edi_status', '=', 'to_send')]"/>
```